Cargo.toml

```diff
@@ -11,31 +11,29 @@ readme = "README.md"
 
 [dependencies]
 ahash = "0.8"
-tokio = { version = "1.43", features = [
+etherparse = { version = "0.17", default-features = false, features = ["std"] }
+log = { version = "0.4", default-features = false }
+rand = { version = "0.9", default-features = false, features = ["thread_rng"] }
+thiserror = { version = "2.0", default-features = false }
+tokio = { version = "1.43", default-features = false, features = [
     "sync",
     "rt",
     "time",
     "io-util",
     "macros",
-], default-features = false }
-etherparse = { version = "0.17", default-features = false, features = ["std"] }
-thiserror = { version = "2.0", default-features = false }
-log = { version = "0.4", default-features = false }
-rand = { version = "0.9", default-features = false, features = ["thread_rng"] }
+] }
 
 [dev-dependencies]
-tokio = { version = "1.43", features = [
-    "rt-multi-thread",
-], default-features = false }
 clap = { version = "4.5", features = ["derive"] }
+criterion = { version = "0.5" } # Benchmarks
+dotenvy = "0.15"
 env_logger = "0.11"
+tokio = { version = "1.43", default-features = false, features = [
+    "rt-multi-thread",
+] }
+tun = { version = "0.7.13", default-features = false, features = ["async"] }
```

Comment:
We can also rename tun2.rs example to tun.rs if you wish

Reply:
ok